pubspec.yaml

```diff
@@ -5,7 +5,7 @@ description: >-
 homepage: https://github.com/dart-lang/shelf
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.6.0 <3.0.0"
```

Comment:
Why did this need to change?

It will need to be changed in `.travis.yml` too if we need to bump the min SDK.

Reply:
I completely forgot why, maybe I removed the bits that required `2.6.0`.